.travis.yml

```diff
@@ -18,6 +18,8 @@ before_install:
 
 install:
     - pip install tox
+    - pip install ordereddict
```

Comment:
Correct.  I had forgotten that I had put this in travis.  I'll remove them and recommit